### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by raboof**
To perhaps make it clearer that https://discuss.akka.io is the place for questions?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/---bug-report.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
 
+---
+
+<!--
 Please report issues regarding specific projects in their respective issue trackers, e.g.:
- - Akka HTTP: https://github.com/akka/akka-http/issues 
- - Alpakka:   https://github.com/akka/alpakka/issues 
+ - Akka-HTTP: https://github.com/akka/akka-http/issues
```
Comment (patriknw) [inline .github/ISSUE_TEMPLATE/---bug-report.md]: do we write it like that now?

File: .github/ISSUE_TEMPLATE/---feature-request.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+<!--
+Please report issues regarding specific projects in their respective issue trackers, e.g.:
+ - Akka-HTTP: https://github.com/akka/akka-http/issues
+ - Alpakka:   https://github.com/akka/alpakka/issues
+ - Akka Persistence Cassandra Plugin: https://github.com/akka/akka-persistence-cassandra/issues
+ - ...
+
+Please explain your issue precisely, and if possible provide a reproducer snippet (this helps resolve issues much quicker).
```
Comment (patriknw) [inline .github/ISSUE_TEMPLATE/---feature-request.md]: This sounds more relevant for issue template. Perhaps something like "Please explain your problem and use case precisely."

File: .github/ISSUE_TEMPLATE/---bug-report.md (hunk 1 above)
Comment (raboof) [reply .github/ISSUE_TEMPLATE/---bug-report.md]: No, that was not intentional, weird

File: .github/ISSUE_TEMPLATE/---feature-request.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+<!--
+Please report issues regarding specific projects in their respective issue trackers, e.g.:
+ - Akka HTTP: https://github.com/akka/akka-http/issues
+ - Alpakka:   https://github.com/akka/alpakka/issues
+ - Akka Persistence Cassandra Plugin: https://github.com/akka/akka-persistence-cassandra/issues
+ - ...
+
+Please explain your use case precisely, and if possible provide an example snippet.
```
Comment (patriknw) [inline .github/ISSUE_TEMPLATE/---feature-request.md]: good

**Review by patriknw — APPROVED** [review]
LGTM

**Review by chbatey — APPROVED** [review]
LGTM

**Pull request merged**